plugin/audit_log/audit_log.c

```diff
@@ -906,6 +926,29 @@ my_bool audit_log_update_thd_local(MYSQL_THD thd,
     const struct mysql_event_general *event_general=
       (const struct mysql_event_general *) event;
 
+    if (event_general->event_subclass == MYSQL_AUDIT_GENERAL_STATUS)
+    {
+      local->skip_query= FALSE;
```

Comment:
I like direct assignments of bool-like vars, i.e. here

local->skip_query= (first if condition)
local->skip_query|= (second if condition)

but it's matter of taste